nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/PackageManagerImpl.java

```diff
@@ -427,6 +426,30 @@ public void registerSource(PackageSource source, boolean local) {
     }
 
     @Override
+    public List<String> listInstalledPackagesNames(PackageType pkgType) {
+        List<DownloadablePackage> upgrades = listInstalledPackages();
```

Comment:
Just about variable naming: `upgrades` is not consistent with the method name and its behavior.